runtime/shibuya/src/lib.rs

```diff
@@ -248,7 +249,12 @@ parameter_types! {
     pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
         .base_block(BlockExecutionWeight::get())
         .for_class(DispatchClass::all(), |weights| {
-            weights.base_extrinsic = ExtrinsicBaseWeight::get();
+            // Adjusting the base extrinsic weight to account for the additional database
+            // read introduced by the `tx-pause` pallet during extrinsic filtering.
+            //
+            // TODO: This hardcoded addition is a temporary fix. Replace it with a proper
+            // benchmark in the future.
+            weights.base_extrinsic = ExtrinsicBaseWeight::get().saturating_add(RocksDbWeight::get().reads(1));
```

Comment:
Only one comment - try & use the general `DbWeight` from the `frame_system::Config`.

RocksDb usage is defined only in one place, and it should remain like that.

Reply:
Approved!